ApiListener/ApiCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ApiListener
```

Comment:
I would like to see if we can find a better solution to sharing ApiListener between BidCat and MusicCat rather than copy-pasting the project. Let me know if you think of a way.